include/GdNeutronHPCapture.hh

```diff
@@ -20,6 +20,14 @@
 #include "G4NeutronHPChannel.hh"
 #include "G4HadronicInteraction.hh"
 
+#include "G4SystemOfUnits.hh"
```

Comment:
seems to be duplicated in https://github.com/WCTE/WCSim/pull/7. for next time, should try to consolidate common changes in a single PR to be merged first, and base the rest on it.

Reply:
resolved in https://github.com/WCTE/WCSim/pull/10

@kmtsui please write this kind of message when resolving specific comments in another PR